pt-BR/mail/chrome/messenger/subscribe.properties

```diff
@@ -35,9 +35,9 @@
 #
 # ***** END LICENSE BLOCK *****
 
-subscribeLabel-nntp=Inscrever os newsgroups selecionados:
+subscribeLabel-nntp=Inscrever os grupos de notícias selecionados:
```

Comment:
CORRECTIONS NEEDED it should be subscribeLabel-nntp=Me inscrever os grupos de notícias selecionados: